app/src/main/java/com/github/se/travelpouch/model/activity/ActivityViewModel.kt

```diff
@@ -0,0 +1,128 @@
+package com.github.se.travelpouch.model.activity
+
+import android.content.Context
+import android.util.Log
+import android.widget.Toast
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.github.se.travelpouch.model.Location
+import com.google.firebase.Firebase
+import com.google.firebase.Timestamp
+import com.google.firebase.firestore.firestore
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+
+class ActivityModelView(val activityRepositoryFirebase: ActivityRepository) : ViewModel() {
```

Comment:
there is a typo : it is ActivityViewModel, no ?

Reply:
Yes, I inverted the two. Also, I will add some documentation